BookingsApi/BookingsApi.UnitTests/Infrastructure.Services/EventDtoMappersTests.cs

```diff
@@ -48,9 +48,8 @@ public void should_map_hearing_confirmation_dto_with_judiciary_participant(Judic
         mappedParticipant.UserRole.Should().Be(role);
     }
     
-    [TestCase(CaseRoleGroup.Judge)]
-    [TestCase(CaseRoleGroup.StaffMember)]
-    public void should_map_hearing_confirmation_dto_with_participant(CaseRoleGroup hearingRoleCode)
+    [Test]
+    public void should_map_hearing_confirmation_dto_with_participant()
```

Comment:
could we rename to `should_map_hearing_confirmation_dto_with_judge_participant()`, now that it only maps the judge

Reply:
done